packages/apify/src/internals/charging.ts

```diff
@@ -0,0 +1,310 @@
+import log from '@apify/log';
+import { Dataset, KeyValueStore } from '@crawlee/core';
+import { type ActorRunPricingInfo, ApifyClient } from 'apify-client';
+
+import { Configuration } from '../configuration.js';
+
+/**
+ * Handles pay-per-event charging.
+ *
+ * @internal
+ */
+export class ChargingManager {
+    readonly LOCAL_CHARGING_LOG_DATASET_NAME = 'charging_log';
+    readonly PLATFORM_CHARGING_LOG_DATASET_ID_KEY = 'CHARGING_LOG_DATASET_ID';
```

Comment:
This feels like it should come from apify/consts

Reply:
Yeah, I admit I have deep disdain for that package. But I honestly believe that it is justified not to use it here since this is all very adhoc and may get removed. Might be a reason to make these private, huh...